LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2018 Jérémie Astori
+Copyright (c) 2015 Jérémie Astori
```

Comment:
I guess this was not intentional

Reply:
Thanks for noticing, @andreineculau!
It was intentional actually, and the reason is the same as mentioned here: https://github.com/thelounge/deb-lounge/pull/16

Reply:
You say there that you believe in bumping the copyright years when the project is updated. You have already done so in January 2018 for this project. Here you reversed the change. Anyways, i was nitpicking while looking at the actual jsonlint disabling

Reply:
Right, I meant specifically xPaw's comment:

> My understanding is that copyright years shouldn't be updated. https://stackoverflow.com/a/2391555/2200891

So most likely I was wrong, and corrected my wrong here :)

Yeah, disabling `jsonlint` isn't great, but npm...